recipes/xnatpy/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set name = "xnat" %}
+{% set version = "0.3.8" %}
+{% set sha256 = "7f4d87b4a2a7df3ce98a0954a418b2d474db4f1c45d40d6a821f6ed8706eda3b" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  noarch: python
+  number: 0
+  script: python -m pip install --no-deps --ignore-installed .
+
+requirements:
+  build:
+    - python
+    - setuptools
```

Comment:
you can remove setuptools here

Reply:
Fair enough ... is there a tutorial/canonical example for writing a `meta.yaml` that you could point me towards? Because there are examples ([e.g.](https://conda.io/docs/user-guide/tasks/build-packages/define-metadata.html#)) which do explicitly list `setuptools`.